tests/redux/actions/global_actions.test.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {browserHistory} from 'utils/browser_history';
+
+import {redirectUserToDefaultTeam} from 'actions/global_actions.jsx';
+
+describe('actions/global_actions', () => {
+    test('redirectUserToDefaultTeam', async () => {
+        browserHistory.push = jest.fn();
+        await redirectUserToDefaultTeam();
+        expect(browserHistory.push).toHaveBeenCalledWith('/select_team');
```

Comment:
I wanted to add a more comprehensive test here but it's hard to mock the store when this action isn't a proper redux action yet